docs/usage/getting-started/running.md

```diff
@@ -92,7 +92,8 @@ WhiteSource Renovate On-Premises and WhiteSource Remediate both run as long-live
 
 ### Global config
 
-Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters. Renovate autodetects a global config file named `config.js`. Other file types and names (`*.js`, `*.json`, `*.json5`, `*.yaml`, `*.yml`) are supported, when you set the `RENOVATE_CONFIG_FILE` environment variable (e.g. `RENOVATE_CONFIG_FILE=config.yaml`).
+Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters. Renovate autodetects a global config file named `config.js`.
```

Comment:
```suggestion
Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters.
By default Renovate checks if a file named `config.js` if present.
```

Reply:
In my opinion this misses the point of documenting the RENOVATE_CONFIG_FILE variable.

Reply:
I'm confused:
1. My suggestion is to add a new line, according to our markdown convention
2. There's still the sentence below

Reply:
Ahh got you. For some reason I was not able to commit GitHub review suggestions on the spot 😄